README.md

```diff
@@ -22,53 +21,12 @@ def draw():
 
 start_p5(setup, draw)
 ```
+### [Documentation](https://github.com/berinhard/pyp5js/blob/master/docs/index.md)
 
-### Installation
-
-This project requires Python 3 and is now on PyPI, so you can install it with `pip` or `pip3`, depending on your environment:
-
-```
-$ pip install pyp5js
-```
-
-### Usage
-
-Since you'll be writting Python code and then generating the correspondent P5.js code from it, pyp5js provides a simple command line API to help you to generate the files.
-
-So, to start a new sketch, you'll have to run:
-
-```
-$ pyp5js new my_sketch
-```
-
-This command will create a directory with the following code structure:
-
-```
-~ my_sketch/
-  ~ static /
-    - p5.js
-  - index.html
-  - my_sketch.py
-```
-
-The `index.html` is prepared to display your sketch, so you'll have to keep on opening it to refresh see results from the code you'll add to `my_sketch.py`.
-
-After updating your code, you'll have to run the `transcrypt` command to update the files. Run it as:
-
-```
-$ pyp5js transcrypt my_sketch
-```
+- [Examples](https://berinhard.github.io/pyp5js/examples/)
 
-The way the code is implemented, for now, you'll have to execute it from the same directory-level where `my_sketch` is - you can't run it from inside the `my_sketch` directory.
+#### [Installation](https://github.com/berinhard/pyp5js/blob/master/docs/index.md#installation)
 
-If you're lazy as me, you can use the `monitor` command instead of the previous one. The command will monitor your sketch directory and keep track of any changes on any `.py` files. When it notices a new change, it automatically runs the transcrypt process for you. So, now you'll just have to refresh your `index.html` file to see the results.
+#### [Usage](https://github.com/berinhard/pyp5js/blob/master/docs/index.md#usage)
 
-```
-$ pyp5js monitor my_sketch
-```
-
-All of the command-line interface methods have a few optional arguments and you can check them by running:
-
-```
-$ pyp5js --help
-```
+#### [Known issues and differences to the Processing.Py and P5.js ways of doing things](known-issues-and-differences-to-the-processingpy-and-p5js-ways-of-doing-things)
```

Comment:
Same think with the other links. It should point to https://berinhard.github.io/pyp5js/#known-issues-and-differences-to-the-processingpy-and-p5js-ways-of-doing-things

Reply:
I think now I fixed it!